boto3/resources/factory.py

```diff
@@ -133,9 +133,10 @@ def load_from_definition(self, resource_name,
 
         base_classes = [ServiceResource]
         if self._emitter is not None:
-            self._emitter.emit('creating-resource-class.%s' % cls_name,
-                               class_attributes=attrs,
-                               base_classes=base_classes)
+            self._emitter.emit(
+                'creating-resource-class.%s' % cls_name,
+                class_attributes=attrs, base_classes=base_classes,
+                service_context=service_context, resource_name=resource_name)
```

Comment:
Do we need the ``resource_name``? Can't we just add it to the constructor of ``CustomModeledAction`` and inject it as an argument to ``create_delete_tags_action`` or pull it off the event name?